Sources/TypeAlias.swift

```diff
@@ -7,13 +7,13 @@
 #if os(OSX)
   public typealias Controller = NSViewController
 
-  func openURL(URL: NSURL) {
-    NSWorkspace.sharedWorkspace().openURL(URL)
+  func openURL(_ url: URL) {
```

Comment:
Putting the _ seems to be the default everywhere tho. Which is a bit stupid, since, if everyone needs underscore, why don't we delete the underscore.